src/main/java/edu/uw/cse403/Account.java

```diff
@@ -0,0 +1,22 @@
+package edu.uw.cse403;
+
+public class Account {
+    double principal,rate; int daysActive,accountType;
+    public static final int STANDARD = 0, BUDGET = 1, PREMIUM = 2, PREMIUM_PLUS = 3;
+
+    public static double calculateFee(Account[] accounts)
```

Comment:
Instead of using floating point which can be inaccurate, let's return the value in cents in integer to be exact.